distributed/src/main/scala/asobu/distributed/EndpointsRegistry.scala

```diff
@@ -0,0 +1,139 @@
+package asobu.distributed
+
+import akka.actor._
+import akka.cluster.{Member, MemberStatus, Cluster}
+import akka.cluster.ddata.{LWWMap, LWWMapKey, DistributedData}
+import akka.cluster.ddata.Replicator._
+import asobu.distributed.EndpointsRegistry.DocDataType
+import play.api.libs.json.{JsNumber, Json, JsObject}
+import concurrent.duration._
+
+trait EndpointsRegistry {
+  val EndpointsDataKey = EndpointsRegistry.EndpointsDataKey
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Field name 'EndpointsDataKey' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/iHeartRadio/asobu/file/2321048327/issues/source?bid=3037259&fileBranchId=3178877#l12)